devicon.json

```
@@ -7827,6 +7827,28 @@
         "color": "#1697F6",
         "aliases": []
     },
+    {
+        "name": "vyper",
+        "altnames": [],
```

I would add vyperlang here since it's commonly used to make it more searchable, and also since their [github](https://github.com/vyperlang/vyper), [twitter](https://twitter.com/vyperlang), [gitter](https://gitter.im/vyperlang/community?at=5ede6d75a3a1b13679d5b527) and [docker](https://hub.docker.com/r/vyperlang/vyper) profiles are named vyperlang: 
```suggestion
        "altnames": ["vyperlang"],
```

---

I'm ok with it!
All done! 👍🏼